src/components/FloatingHelper/HelperContent/HelperContent.tsx

```diff
@@ -46,28 +48,31 @@ export const HelperContent: React.SFC<HelperContentProps> = (
             </Text>
           </div>
         )}
-        {actionText &&
+        {actionText && onAction &&
           actionText.length > 0 && (
             <Button
               className={style.action}
               data-hook={DataHooks.actionButton}
               skin={themeToButtonProps[actionTheme].skin}
               priority={themeToButtonProps[actionTheme].priority}
               size={ButtonSize.small}
+              onClick={onAction}
             >
               {actionText}
             </Button>
           )}
-      </div> 
+      </div>
       {image && <div data-hook={DataHooks.image} className={style.image}>{image}</div>}
     </div>
   );
 };
 
 HelperContent.propTypes = {
   title: string,
-  body: string,
-  actionText: string
+  body: string.isRequired,
+  actionText: string,
+  onAction: func,
```

Comment:
onAction what? :)
Maybe a better name?

Reply:
I was trying to follow WSR api for SectionHelper:
https://github.com/wix/wix-style-react/blob/b5a7c246265475cd799dd1420780e45aad0c237f/src/SectionHelper/index.js#L80

Reply:
onActionClick?